src/url.rs

```diff
@@ -0,0 +1,387 @@
+                    prov:agent <https://data.brreg.no/enhetsregisteret/api/enheter/971277882> ] . 
+                <http://publications.europa.eu/resource/authority/language/ENG> rdf:type dct:LinguisticSystem ; 
+                    <http://publications.europa.eu/ontology/authority/authority-code> "ENG" ; 
+                    skos:prefLabel "Engelsk"@nb . 
+                <http://publications.europa.eu/resource/authority/language/NOR> rdf:type dct:LinguisticSystem ; 
+                    <http://publications.europa.eu/ontology/authority/authority-code> "NOR" ; skos:prefLabel "Norsk"@nb .
+        "#.to_string());
+
+        assert!(mqa_event_result.is_ok());
+        let mqa_event_option = mqa_event_result.unwrap();
+        assert!(mqa_event_option.is_some());
+        let store_actual_result = parse_turtle(mqa_event_option.unwrap().graph);
+        assert!(store_actual_result.is_ok());
+        let store_actual = store_actual_result.unwrap();
+        assert_eq!(
+            8,
+            store_actual
+                .quads_for_pattern(None, None, None, None)
+                .count()
+        );
+
+        let dataset = NamedNodeRef::new_unchecked("https://registrering.fellesdatakatalog.digdir.no/catalogs/971277882/datasets/29a2bf37-5867-4c90-bc74-5a8c4e118572");
+        let mut count = 0;
+
+        for dr in list_distributions(dataset, &store_actual) {
+            count = count + 1;
+
+            if let Ok(dist_quad) = dr {
+                let dist = convert_term_to_named_or_blank_node_ref(dist_quad.object.as_ref());
+                assert!(dist.is_some());
+
+                let mr = store_actual
+                    .quads_for_pattern(
+                        dist.map(|d| d.into()),
+                        Some(dqv::HAS_QUALITY_MEASUREMENT),
+                        None,
+                        None,
+                    )
+                    .next()
+                    .unwrap();
+
+                if let Ok(measurement_quad) = mr {
+                    let measurement =
+                        convert_term_to_named_or_blank_node_ref(measurement_quad.object.as_ref());
+
+                    let vr = store_actual
+                        .quads_for_pattern(
+                            measurement.map(|m| m.into()),
+                            Some(dqv::VALUE),
+                            None,
+                            None,
+                        )
+                        .next()
+                        .unwrap();
+
+                    if let Ok(value_quad) = vr {
+                        info!("{}", value_quad);
+                        match value_quad.object {
+                            Term::Literal(value) => assert_eq!("400", value.value()),
+                            _ => assert!(false),
+                        }
+                    } else {
+                        assert!(false);
+                    }
```

Comment:
Foretrekker unwrap og `assert_eq` fordi det gir mer informasjon enn `assert!(false)` når noe feiler.

```suggestion
                        .unwrap()
                        .unwrap();

                    info!("{}", value_quad);
                    assert_eq!(
                        value_quad.object,
                        Term::Literal(Literal::new_typed_literal(
                            "400",
                            NamedNode::new_unchecked("http://www.w3.org/2001/XMLSchema#integer"),
                        ))
                    );
```